programs/github-pull-request/pull/index.js

```diff
@@ -1,7 +1,17 @@
 const {join} = require('path');
 const GitHub = require('../../../lib/GitHub');
 const {name} = require('../../../package.json');
-const UNIQUE_IDENTIFIER = Buffer.from(name).toString('base64');
+
+/**
+ * Returns the Stats pull request comment identifier (to be later edited if needed)
+ * @param repo
+ * @param projectName
+ */
+const getCommentIdentifier = ({ repo, projectName }) => Buffer.from(projectName ? [
+		name,
+		repo,
+		projectName,
+].filter(Boolean).join('-') : name).toString('base64');
```

Comment:
Why do you need the `projectName ? ...`? Why this not enough?
```suggestion
const getCommentIdentifier = ({ repo, projectName }) => Buffer.from([
    name,
    repo,
    projectName,
].filter(Boolean).join('-')).toString('base64');
```

Reply:
Cause it would mean that i will change the default ( Cases where `projectName` is missing ) to something else than what it is today (solely `name`).

If that happen, once merged, This bot won't be able to edit past comments, and will result with duplicates ( since it won't have that id, so it will create a new comment )